.github/workflows/cd.yml

```diff
@@ -26,14 +26,15 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm ci
     - run: npm run build
+    - run: npm run interop
     - run: npm test
 
 # Push dist/ to dist
     - run: git checkout --orphan dist
     - run: git config --local user.email "f6eae15e-09a6-45d4-b9a1-f694f50b9e21@example.com"
     - run: git config --local user.name "🤖"
     - run: git reset
-    - run: git add -f ./distribution/index.html
+    - run: git add -f ./distribution/index.html ./distribution/tscript.js
```

Comment:
But is it really necessary? The IDE includes its own version of the interpreter and the e-learning system as well.

It would be nice if we could separate this from the IDE deployment — @TGlas, would you be fine with running the build yourself for the e-learning system?

Reply:
Short answer: yes.

Running a separate build for this is fine, since the e-learning system is updated rather infrequently. However, I would appreciate to keep the build script inside of the project, since it should always be up to date.

Is it possible to define dependent targets? In a makefile I would define the IDE as the default target, and have a separate target for `tscript.js`. Other targets could be the unit tests and maybe more things in the future, like a documentation self-test. Finally, an "all" target would be nice to have.

Reply:
You would just need to run `npm run interop` .  In that case there is no point in having it part of the gh-pages branch. But i am for building it anyways to ensure that the build works.

Reply:
> Other targets could be the unit tests and maybe more things in the future, like a documentation self-test.

NPM scripts (defined [here](https://github.com/TGlas/tscript/blob/cfb3fdafc10fa0f4cfe0bfb3171b79fbf0dccfab/package.json#L6-L11)) basically take the role of make targets. Unit tests, for example, are already included in this. It would be possible to run other scripts from within a script using `npm run a && npm run b` or the package [npm-run-all](https://www.npmjs.com/package/npm-run-all).

Reply:
Thanks for clarifying!